wp-security-audit-log/custom-alerts.php

```diff
@@ -252,13 +252,14 @@
             [
                 8820,
                 WSAL_MEDIUM,
-                __( 'User Enabled/Disabled ryte integration in the Yoast SEO plugin settings', 'activity-log-wp-seo' ),
-                __( 'Changed the <strong>Ryte integration</strong> plugin feature.', 'activity-log-wp-seo' ),
+                __( 'User Enabled/Disabled an integration in the Yoast SEO plugin settings', 'activity-log-wp-seo' ),
+                __( 'Changed the %type% plugin feature.', 'activity-log-wp-seo' ),
```

Comment:
This change will change the feature type from bold to regular text. Is it intentional? Perhaps it should be like the following?

```suggestion
                __( 'Changed the <strong>%type%</strong> plugin feature.', 'activity-log-wp-seo' ),
```